app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -896,9 +898,7 @@ class AdvancedFilter extends React.Component {
               </Form.Group>
             )}
           </Col>
-          {filterOption && filterOption.tooltip && filterOption.type !== 'relative' && (
-            <span className="align-middle mx-2">{this.renderOptionTooltip(filterOption, value, index)}</span>
-          )}
+          {filterOption && filterOption.tooltip && <span className="align-middle mx-2">{this.renderOptionTooltip(filterOption, value, index)}</span>}
```

Comment:
Why was the check for the filter type removed?

Reply:
it's not needed because relative filterOptions don't have tooltip defined

Reply:
Yeah but if they did, it would need to be ignored otherwise it'd cause problems. That's actually why I put the check there

Reply:
A tooltip shouldn't ever be defined for a relative fields because then that would mean there is two sets of conflicting tooltip text.  Just seems redundant for how we have things set up now

Reply:
Honestly the way it is not is now is not great anyway, it'd be best if it was really easy to add a tooltip to any advanced filter. I actually made a ticket after making this change to improve this: [SARAALERT-1083](https://tracker.codev.mitre.org/browse/SARAALERT-1083). Either way, I'd prefer we keep the code that checks this just to prevent issues for now. I know no one _should_ be declaring a tooltip for relative fields, but in the case they do why not prevent the issue given how it's currently set up.

Reply:
modified this to just make one call to the render tooltip so it should be cleaner now if we want to add any in the future

Reply:
I think you meant for it to be NOT equal to "custom", yeah? Since we render the other tooltip when it IS custom.
```
filterOption && (filterOption.tooltip || relativeOption !== 'custom')
```

Reply:
nope - I removed the other one so it is only rendered in one place. see here: https://github.com/SaraAlert/SaraAlert/pull/600/commits/9c23d177a901f2a79eb843e71cc9a9f227e2f98a